traits/observers/expression.py

```diff
@@ -76,11 +85,82 @@ def then(self, expression):
         """
         return SeriesExpression(self, expression)
 
+    def dict_items(self, notify=True, optional=False):
+        """ Create a new expression for observing items inside a dict.
+
+        Events emitted (if any) will be instances of
+        :py:class:`~traits.observers.events.DictChangeEvent`.
```

Comment:
The default Sphinx domain is `py`, so we can leave out the `:py` prefix if that makes the docstring more readable.